RUM Auto-instrumentation updates (#28233)
DataDog/documentation: content/en/real_user_monitoring/browser/setup/_index.md

> ---

Reviewer: How is a customer supposed to navigate to this section? Given the path it looks to me like it is supposed to be visible when clicking here

auto-instrumentation-updates/real_user_monitoring/browser/setup
<img width="233" alt="image" src="https://github.com/user-attachments/assets/121ed8d2-c376-44e5-b41f-c24c15f43e1a" />
<img width="740" alt="image" src="https://github.com/user-attachments/assets/a65a7e12-21b7-46f4-b1ab-99f765b0550e" />

But instead on that left menu, or the breadcrumbs, the content for "real_user_monitoring/browser/setup/client" is shown instead. It might just be me misunderstanding how navigation is supposed to happen, though (If so, can you link to the page here? 🙇 ).

title: Browser Monitoring Setup
description: Understand your options for setting up the RUM Browser SDK.
further_reading:
- link: '/real_user_monitoring/browser/'
  tag: 'Documentation'
  text: 'RUM Browser Monitoring'
---

## Overview

To get started with Datadog RUM Browser, you need to:

1. Create an application in Datadog.
2. Instrument your application.

<div class="alert alert-info"><strong>Simplify your setup!</strong> Install the SDK and instrument your application in one step with <a href="https://docs.datadoghq.com/real_user_monitoring/browser/setup/server/">Auto-Instrumentation</a>.</div>

Instrumenting your application allows observability data from your application to be displayed in the Datadog UI.

## Instrumentation types

> There are two main approaches to instrument your application: automatic or manual instrumentation.

Author: ```suggestion
There are two ways to instrument your application: automatic or manual instrumentation.
```

### Auto-instrumentation

{{< callout url="https://www.datadoghq.com/private-beta/rum-sdk-auto-injection/" btn_hidden="true" header="Try the Auto-Instrumentation Preview!">}}

You can set RUM configs on your web servers and Datadog will automatically inject RUM configs to instrument your RUM applications. Learn more about <a href="/real_user_monitoring/browser/setup/server">Auto-Instrumentation.</a>
{{< /callout >}}

Automatically add RUM JS to your web app HTML with minimal steps by installing the RUM injector by downloading the RUM Installer or by CLI (running a command).

> **SRE and engineering teams** without access to frontend code or **teams who need to manage** all observability needs centrally may find this useful for:
> 
> - Unlocking performance data across all applications upon setting up RUM
> - Start holistically monitoring application performance across the organization

Reviewer 2: ```suggestion
**SRE and engineering teams** without access to frontend code, or **teams who need to manage** all observability needs centrally, may find this useful for:

- Unlocking performance data across all applications upon setting up RUM
- Holistically monitoring application performance across the organization
```

To learn more, see [auto-instrumentation][1].

> ### Client-side (manual instrumentation)

Reviewer 2: Earlier, you refer to the two types as "automatic" or "custom" instrumentation. Does client-side = manual = custom? Flagging so we can keep terminology consistent if possible!

Add the RUM SDK manually to your application code.

**Frontend engineering, mobile engineering, or product teams** with access to frontend code may find this method useful for:

> - Daily engineering needs (for example: live support, troubleshooting, and downstream services health checks)

Reviewer 2: ```suggestion
- Daily engineering needs (for example: live support, troubleshooting, and health checks for downstream services)
```

> - Daily engineering needs (live support, troubleshooting, downstream services health checks, and so on)

Reviewer 2: Not sure if "so on" translates well, so small tweak!

```suggestion
- Daily engineering needs (for example: live support, troubleshooting, and downstream services health checks)
```

> - Product needs (user flow analysis, user segmentation, feature flag tracking, and so on)

Reviewer 2: ```suggestion
- Product needs (for example: user flow analysis, user segmentation, and feature flag tracking)
```
